stan/inc2prev_antibodies.stan

```diff
@@ -60,6 +61,7 @@ parameters {
   real<lower = 0> rho; // length scale of gp
   real<lower = 0> alpha; // scale of gp
   vector[M] eta; // eta of gp
+  real inc_init; // Initial infections
```

Comment:
Should this be a vector of length `diff_order` where the first element is initial value of infections, second initial value of growth in infections etc.? See below

Reply:
Yes agree if not comfortable with letting the GP having a slightly warped initial definition. I will do some testing and see how much the model complains as implemented to see if this is needed.

Reply:
This does appear to really make much difference so I would probably suggest leaving as.